DEVELOPMENT_ENVIRONMENT.md

```diff
@@ -97,3 +97,66 @@
 	- **Important:** select "Do not format other file types by IntelliJ formatter"
 	- go to Preferences -> Editor -> Code style -> Java : set class and static names counts for import with * to 99
 	- for Android Studio, code formatting is usually done with Ctrl+Alt+L. For automatic formatting, it's recommended to use the plugin Save Actions (https://plugins.jetbrains.com/plugin/7642-save-actions)
+
+## Enable debugging of the application
+**Note: This is only needed for development of the SORMAS back and front apps in order to enable debugging
+* Install the latest Android Studio version (to avoid errors, start the installation with admin rights)
+* For Intellij -> Download Intellij Idea newer that 15 of April 2020. Version prior to 15 of April is containing a bug which will not allow to enable debugging
+* For Payara -> Better versions which do support debugging integration as today (14.01.2020) - is 5.192 (this version is set as well in the production)
+* For Payara is needed to be replaced launcher.jar from Payara launcher.zip. For that:
+	- Payara server should be stopped
+	- go to https://youtrack.jetbrains.com/issue/IDEA-216528
+	- download launcher.jar file attached to the conversation(at least for windows PCs it will be downloaded as a ZIP file. So please rename it by writing JAR instead of ZIP)
+	- need to be deleted the cash from opt\domains\sormas\osgi-cache\felix (folder felix should be deleted as well)
+	- replace original launcher.jar file with downloaded (and renamed one) for next 2 paths: 1) opt\payara5\glassfish\launcher.jar and opt\payara5\glassfish\modules\launcher.jar
+* Only after fixing those issues, DEBUG should start well.
+
+## Issues which were found during installation process of the project
```

Comment:
This is quite tedious. When we have specific defects in the setup, then we could have a headline at the end of the file and in parallel enhance the automation (tickets to be planned in).
Documentation should not mention persons but state facts.